packages/sns/src/types/governance.params.ts

```diff
@@ -39,3 +39,19 @@ export interface SnsDisburseNeuronParams {
   neuronId: NeuronId;
   amount?: E8s;
 }
+
+/**
+ * The parameters to set dissolve timestamp
+ */
+export interface SnsSetDissolveTimestampParams {
```

Comment:
for the state of the art, can/should the params extends `Omit<QueryParams, "certified">` in case we someday add more params in `QueryPrams`?

Reply:
Nevermind, we don't do it for the other params. Maybe we should add something to strengthness these multiple functions that are commented with `// Always certified`

Reply:
being said, all `interface` of this file uses `neuronId: NeuronId;`, maybe, again not in this PR, we can refactor this too

Reply:
Good point, will be done in another PR 👍